packages/react-instantsearch/src/core/highlight.test.js

```diff
@@ -1,101 +1,112 @@
 /* eslint-env jest, jasmine */
-import React from 'react';
+import {parseAlgoliaHit, parseHighlightedAttribute} from './highlight.js';
```

Comment:
There should be few cases where testing a "private" method (here parseHighlightedAttribute) should be really necessary. By private I mean a function only used in one file. I would prefer not having to expose or test this function in isolation and rather only test the wrapping method which is parseAlgoliaHit.

parseHighlightedAttribute is not used outside of highlight.js. parseAlgoliaHit is checking for bad usage of parseHighlightedAttribute.

Exporting this function and testing it makes it harder to refactor the code in highlight.js: you cannot tell easily if parseHighlightedAttribute is part of the API or used by other files, you will have to search in all files to understand that it's exported to be able to test it.